packages/medusa/src/migrations/1674455083104-product_category_product.ts

```diff
@@ -0,0 +1,47 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class productCategoryProduct1674455083104 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `
+        CREATE TABLE "product_category_product" (
+          "id" character varying NOT NULL,
+          "product_category_id" character varying NOT NULL,
+          "product_id" character varying NOT NULL,
```

Comment:
todo: Add foreign key constraints to `"product"("id")` and `"product_category"("id")`